src/objects/zcl_abapgit_objects_super.clas.abap

```diff
@@ -308,11 +308,12 @@ CLASS ZCL_ABAPGIT_OBJECTS_SUPER IMPLEMENTATION.
     ls_e071_tadir-obj_name = ms_item-obj_name.
     INSERT ls_e071_tadir INTO TABLE lt_e071_tadirs.
 
-    CALL FUNCTION 'RS_INACTIVE_OBJECTS_WARNING'
+CALL FUNCTION 'RS_INACTIVE_OBJECTS_WARNING'
```

Comment:
run pretty printer, indentation wrong, also `phased_activation` is a optional parameter, so no need to add it

Reply:
done